src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -486,6 +489,119 @@ trait BronzeTransforms extends SparkSessionWrapper {
     }
   }
 
+  def prepClusterSnapshot(
+                           workspace: Workspace,
+                           pipelineSnapTS: TimeTypes,
+                           apiEnv: ApiEnv,
+                           database: Database,
+                           clusterSnapshotErrorsTarget: PipelineTable,
+                           config: Config,
+                           apiEndpointTempDir: String
+                         )(auditDF: DataFrame) : DataFrame = {
+
+    val clusterIDs = auditDF
+      .select(getClusterIDsFromAudit.alias("cluster_id"))
+      .filter(col("cluster_id").isNotNull)
+      .distinct()
+      .select("cluster_id").distinct().collect().map(x => x(0).toString)
```

Comment:
you're already doing a select("cluster_id") and a distinct right above, we can remove the duplicate calls

Reply:
Done